__tests__/components/ui/infoicon.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { InfoIcon } from "@/components/ui/infoicon.tsx";
+
+describe("InfoIcon", () => {
+  const iconSize = 16;
+  const content = "This is a test string for the tooltip content.";
+
+  it("renders an InfoIcon button", () => {
+    render(<InfoIcon iconSize={iconSize} content={content} />);
+    const infoIconButton = screen.getByRole("button");
+    expect(infoIconButton).toBeInTheDocument();
+  });
+
+  it("displays tooltip content on focus", () => {
+    render(<InfoIcon iconSize={iconSize} content={content} />);
+    const infoIconButton = screen.getByRole("button");
```

Comment:
Readability nit: tests usually have parts that one can leave blank lines between to make them easier to read: setup, execute, & verify. The lines above are the setup, please add a blank line after them.